open_seq2seq/encoders/tdnn_encoder.py

```diff
@@ -28,6 +28,8 @@ def get_optional_params():
         'bn_momentum': float,
         'bn_epsilon': float,
         'use_conv_mask': bool,
+        'drop_block_prob': float,
```

Comment:
We should document how to use these settings in the init.
I should also document what use_conv_mask is.

Reply:
drop_block_prob controls the probability of dropping blocks on training, drop_block_index defines which block to drop on inference. Thus, the default parameters which will correspond to turned off BlockOut are drop_block_prob=0.0, drop_block_index=-1